apcd-cms/src/apps/admin_exception/templates/edit_exception_modal.html

```diff
@@ -0,0 +1,127 @@
+                    inputmode="numeric"/>
+                    <div id="help-text-approved" class="help-text">
+                      Current: {{r.approved_expiration_date}}
+                    </div>
+                </div>
+                <div class="field-wrapper textinput required">
+              <label for="status">
+                Exception Status
+              </label>
+              <select name='status'  class="choicefield" id='status'>
+                {% for option in status_options %}
+                {% if option != 'All' %}
+                <option class="dropdown-text" {% if option == r.status %}selected{% endif %}>{{ option }}</option>
+                {% endif %}
+                {% endfor %}
+              </select>
+            </div>
+            <div class="field-wrapper textinput required">
+              <label for="status">
+                Exception Outcome
+              </label>
+              <select name='outcome' class="choicefield" id='outcome'>
+                {% for option in outcome_options %}
+                <option class="dropdown-text" {% if option == r.outcome %}selected{% endif %}>{{ option }}</option>
+                {% endfor %}
+              </select>
+            </div>
+            </div>
+            <div class="field-wrapper textarea required">
+            <label for="notes">
+              Exception Notes
+            </label>
+
+              <div class="field-errors" style="display: none"></div>
+                <textarea name="notes" cols="40" rows="5" class="textinput" type="text" id="notes"
+                  minlength="2" maxlength="2000">{{ r.notes }}</textarea>
+                <div class="help-text">
+                  2000 character limit
+                </div>
+              </div>
+            <div class="button-wrapper submit">
+            <button class="form-button" type="submit" value="Submit">
+              Submit
+            </button>
+          </div>
+          <hr>
+        </form>
+      </div>
+              <div class="modal-section">
+                <h4 class="modal-header">Current Exception Information</h4>
+                  <dl class="c-data-list--is-vert c-data-list--is-wide">
+                    <dt class="c-data-list__key">Created</dt>
+                    <dd class="c-data-list__value">{{r.created_at}}</dd>
+                    <dt class="c-data-list__key">Organization</dt>
+                    <dd class="c-data-list__value">{{r.org_name}}</dd>
+                    <dt class="c-data-list__key">Requestor</dt>
+                    <dd class="c-data-list__value">{{r.requestor_name}}</dd>
+                    <dt class="c-data-list__key">Requestor Email</dt>
+                    <dd class="c-data-list__value">{{r.requestor_email}}</dd>
+                    <dt class="c-data-list__key">Exception Type</dt>
+                    <dd class="c-data-list__value">{{r.exception_type}}</dd>  
```

Comment:
```suggestion
                    <dd class="c-data-list__value">{{r.request_type}}</dd>  
```
Aligning this with var defined in view.